package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-filings-ui",
-  "version": "5.0.0",
+  "version": "5.0.1",
```

Comment:
Please run `npm i` to update this file. Did you update it manually instead? It is missing some changes.